vhdl_lang/src/ast/visitor.rs

```diff
@@ -1314,26 +1292,13 @@ impl ASTNode for LabeledSequentialStatement {
     }
 }
 
-impl ASTNode for SubprogramSpecification {
```

Comment:
Blocking: The ASTNode impl for SubprogramSpecificafion is still need to visit the children.  My earlier comment was just about removing `visit_subprogram_specification´

Reply:
Hmm I guess I understood you wrong. I re-did the commit, kept the `ASTNode` impls, but removed the calls to `visitor.visit_*` and replaced them with a hardcoded `VisitorResult::Continue`